tempo/kfserving/k8s.py

```diff
@@ -34,10 +33,15 @@
 }
 
 
-class KFServingKubernetesRuntime(Runtime, Remote):
+class KFServingOptions(RuntimeOptions):
```

Comment:
Oh I thought the objective was to change it to `KubernetesOptions` instead, with an "orchestrator" selected as a sub-option instead, Could this not be the higher level abstraction? Something like:
```
class Options(BaseModel):
   runtime: str = "tempo.seldon.SeldonDockerRuntime"
   orchestrator: str = "seldon-core"


class KubernetesOptions(Options):
   runtime: str = "tempo.seldon.SeldonKubernetesRuntime"
   orchestrator: str = "seldon-core"
   replicas: int = 1
   namespace = "default"
   minReplicas: Optional[int] = None
   maxReplicas: Optional[int] = None
   authSecretName: Optional[str] = None
   serviceAccountName: Optional[str] = None
   ingress_options: IngressOptions = IngressOptions()


class SeldonDeployAuthType(Enum):
   session_cookie = "session_cookie"
   oidc = "oidc"


class SeldonEnterpriseOptions(KubernetesOptions):
   runtime: str = "tempo.seldon.SeldonDeployRuntime"
   orchestrator: str = "seldon-core"
   host: str
   user: str
   password: str
   auth_type: SeldonDeployAuthType = SeldonDeployAuthType.session_cookie
   oidc_client_id: str
   oidc_server: str
   verify_ssl: bool = True
```